docker-local/wallet-events-producer-testing/data.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
```

Comment:
This is a sample data for payment-wallets collection:

`{
  "_id": "00a06383-1495-4b90-88f6-80c5fecf554a",
  "userId": "928bbaed-f7c2-41f9-b47c-9bce088322d6",
  "status": "VALIDATED",
  "paymentMethodId": "148ff003-46a6-4790-9376-b0e057352e45",
  "contractId": "W171896465077471gI",
  "applications": [
    {
      "_id": "PAGOPA",
      "status": "ENABLED",
      "creationDate": "2024-06-21T10:10:50.749146096Z",
      "updateDate": "2024-06-21T10:10:50.749146096Z",
      "metadata": {
        "onboardByMigration": "2024-06-21T10:10:50.749146096Z"
      }
    }
  ],
  "details": {
    "type": "CARDS",
    "bin": "223059",
    "lastFourDigits": "4353",
    "expiryDate": "203204",
    "brand": "MASTERCARD",
    "paymentInstrumentGatewayId": "41795763745a7246744943674e6265776e6a50674c375555754447687534386b4a7574706a5975307030593d",
    "_class": "it.pagopa.wallet.documents.wallets.details.CardDetails"
  },
  "clients": {
    "IO": {
      "status": "ENABLED"
    }
  },
  "version": 2,
  "creationDate": "2024-06-21T10:10:50.802632659Z",
  "updateDate": "2024-06-21T10:34:18.850935581Z",
  "onboardingChannel": "IO",
  "_class": "it.pagopa.wallet.documents.wallets.Wallet"
}`

[Here](https://github.com/pagopa/pagopa-payment-wallet-scheduler-service/pull/4/files#diff-4145e993a33209413e12c383914fcf689f922f49bf03455e555657f9561a8ee2) you can see the query used for wallets search.
Currently the status of the search wallet is “VALIDATED”, for me it is recommended to also enter wallets in a different status to verify the validity of the query

Reply:
done here 8afa8fc4433641cde88a4495e2b402b0e2202b30